ert_shared/ensemble_evaluator/prefect_ensemble/unix_step.py

```diff
@@ -0,0 +1,144 @@
+import os
+import subprocess
+from prefect import Task
+from ert_shared.ensemble_evaluator.prefect_ensemble.client import Client
+from ert_shared.ensemble_evaluator.prefect_ensemble.storage_driver import (
+    storage_driver_factory,
+)
+from cloudevents.http import to_json, CloudEvent
+from ert_shared.ensemble_evaluator.entity import identifiers as ids
+
+
+class UnixStep(Task):
```

Comment:
Could this class be tested by itself? Think prefect allow you to execute run manually on a task.

Reply:
I will look into it